mod/wikirate_source/set/type/source/source.rb

```diff
@@ -36,26 +38,20 @@ def indirect_contributor_search_args
 end
 
 def has_file_or_text?
-  file_card = subcards["+File"]
-  text_card = subcards["+Text"]
-  ( file_card && (file_card.stringify_keys.has_key?("file") || file_card.stringify_keys.has_key?("remote_file_url"))) || 
-  ( text_card && ( text_card_content = (text_card.stringify_keys)["content"] ) && !text_card_content.empty? )
+  file_card = subfield(:file)
+  text_card = subfield(:text)
+  ( file_card && file_card.attachment.present? ) || 
+  ( text_card && text_card.content.present? )
```

Comment:
Use 2 (not 0) spaces for indenting an expression spanning multiple lines.<br>Space inside parentheses detected.